README.md

```diff
@@ -67,3 +68,6 @@ Have a look at our jupyter notebook:
   year={2019}
 }
 ```
+
+see [here](https://github.com/mpicbg-csbd/structured_N2V_paper) for citing `StructN2V`.
```

Comment:
This link returns a 404 for me.

Reply:
hmmm not for me. wonder what's going on.

Reply:
the _upper_ link `https://github.com/juglab/n2v/tree/structn2v/examples/2D/structN2V_2D_synth_mem` gives 404, but i guess that's because the we haven't merged yet ;)

Reply:
hmm. this one works for you? https://github.com/mpicbg-csbd/structured_N2V_paper